app/ui/src/app/customizations/customizations.module.ts

```diff
@@ -35,7 +35,7 @@ const routes: Routes = [
     ]
   // TODO: Move to its own NgRoutingModule
   }, {
-    path: 'tech-extensions/import/:id',
+    path: 'tech-extensions/import/:id/:extensionId',
```

Comment:
I'm not keen on putting the `extensionId` field in the URL when we already have `extension.id` present. I've implemented a different option in #860 which I think does the same as this but without changing the path. Really happy to listen to you @deeleman on which way you think is best to go though, let's pick and commit :)

Reply:
Issue here is that we need both properties: `id` is required for fetching the upload URL and `extensionId` is required for display purposes. We could fetch both by using querying the backend with any of them but this would require more overhead.

I am not familiar with the internals of the associated store, but if you feel that by passing `extensionId` to `this.extensionStore.load(...)` you'll get the information you require, then go ahead.

Reply:
I'd personally prefer loading the details from the server if they are not already cached. Considering we also need the name, etc this seems to make sense to me rather than adding more info to the URL.

Reply:
Either that or introduce a little service to hold the data required and use the data that we've fetched from the list page.  

The issue is that the update/install page needs the extensionId field to determine whether to show the 'install' or 'update' version of the page, I didn't personally want to make another call just to get this data when we've already fetched it from the list page.

To be honest, I figure this code is going to be reworked once we move this section of the app away from using the restangular store and service.

Reply:
meh, I guess the logic would be a lot more consistent if we just fetch and only keep the ID in the URL, so go for it :-)